### DISCUSSION TITLE
New App Info Header

### DISCUSSION THREAD

**Discussion by Iamlooker** [Droid-ify/client#30] [Show and tell]
# Which one do you prefer?

## New Version

![Screenshot_20220710-023504_Droid-ify-Debug](https://user-images.githubusercontent.com/60404093/178122690-23509c6c-b5f9-4e59-a72a-4a036b4ae9d8.png)

## Old Version

![Screenshot_20220710-023726_Droid-ify](https://user-images.githubusercontent.com/60404093/178122693-25cbc060-c376-4568-b229-1a234e02855a.png)

**Comment by MysteriousOrb2000** [comment]
Old one looks better and is more usable due to spacing.

**Comment by Jrchintu** [comment]
Old one, since it also aligns with the playstore. And old one saves space & hence scrolling (no one would love unnecessary scrolling).

**Comment by BraINstinct0** [comment]
~Old one.~ Edit: a tie.
1) Compactness - Efficient use of space while not being cluttered
2) Alignment - Center alignment doesn't quite look good (IMO) on wide-width devices. Especially those like Android TV, Android tablets, etc.

**Reply by Iamlooker** [reply]
I have a separate idea for large screen devices to make them more usable.

But I am currently working on renovating the database, the app will surely get a TV and tablet support

**Reply by BraINstinct0** [reply]
Hmm, tbh both looks good for narrow-width devices so I'll change my opinion to a tie.

**Comment by HikaruIchijyo** [comment]
New one😬🫣

**Comment by Donkey-Doug** [comment]
Hi, what about removing the background (see https://github.com/Iamlooker/Droid-ify/issues/141)
